package.json

```diff
@@ -111,4 +116,4 @@
     "tsconfig-paths": "4.1.1",
     "typescript": "^4.7.4"
   }
-}
+}
```

Comment:
Don't want to remove that trailing newline.

Reply:
I think we're gonna need to rethink the method we use to update the `package.json`.

Reply:
Did you look any further into that npm built-in command? Might be useful here.

Reply:
My original thought was maybe a script that runs that command thats built in but i was under the impression that wasnt what you wanted.